queries/landuse.jinja2

```diff
@@ -29,19 +48,27 @@ WHERE
 UNION ALL
 
 SELECT
-    name,
-    {{ ne_landuse_cols('park or protected land') }}
+  name,
+  operator,
+  way_area::bigint AS area,
+  COALESCE("boundary", "landuse", "leisure", "natural", "highway", "aeroway", "amenity", "tourism", "man_made", "power") AS kind,
```

Comment:
Adding it at the end might not have the effect that you want; many of these `boundary=national_park` areas also have `landuse`, `leisure` or `natural` tags, and so the `boundary` value would be shadowed by these other values. This would mean that most `national_park` areas would be indistinguishable from other areas. That's why I added it at the beginning.

Using `COALESCE` like this is basically dropping a whole bunch of useful information, and the side-effect is that the values in the `COALESCE` need to be sorted by "importance". Any key that appears before another must be more important than that other, and if the first has a value then the second's value (or lack of it) will be ignored.

If we hide the `national_park` value at the back of the list, it's likely that very few polygons will get this tag. Let me know if that's still the preferred option.